n4js-libs/packages/n4js-cli/package.json

```diff
@@ -18,6 +18,7 @@
   "files": [
     "bin/n4jsc.js",
     "bin/n4jsc.jar",
+    "bin/PrintJavaVersion.java",
```

Comment:
It's confusing to have Java and Javascript source files in the `bin` folder.

Since the `src` folder already contains a sub-folder `n4js` which contains the N4JS sources, it would be best to create two sub-folders `js` and `java` below folder `src` and place those files there. I think this is then in line with the structure of other n4js-libs and N4JS code bases.

Also, `bin` folders in npm packages are often used for binaries that may make sense to be used directly. This applies to the JRE and the n4jsc.jar, but not the .js and .java files.